backend/experiment/rules/matching_pairs_2025.py

```diff
@@ -55,15 +59,151 @@ def next_round(self, session):
             score = Final(
                 session,
                 title="Score",
-                final_text=self.final_text(session),
+                final_text=self._final_text(session),
                 button={"text": "Play again", "link": self.get_play_again_url(session)},
                 rank=self.rank(session, exclude_unfinished=False),
                 feedback_info=feedback_info,
                 percentile=session.percentile_rank(exclude_unfinished=False),
             )
             return [score]
 
-    def final_text(self, session):
+    def get_matching_pairs_trial(self, session):
+        player_sections = self._select_sections(session)
+        random.seed(self.random_seed)
```

Comment:
Is this for testing purposes?

Reply:
No, I renamed the `final_text` method to `_final_text` and `git diff` got confused